src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -311,9 +311,9 @@ private static Object deleteUser(Request req, Response res) {
                     for (String targetId : sub.getTarget()) {
                         FeedSource fs = Persistence.feedSources.getById(targetId);
                         if (fs == null) continue;
-
+                        boolean isProjectAdmin = userProfile.canAdministerProject(fs.projectId);
                         // FeedSource comments
-                        for (Note note : fs.retrieveNotes()) {
+                        for (Note note : fs.retrieveNotes(isProjectAdmin)) {
```

Comment:
if a user is an admin, leaves admin-only comments, and then is demoted to a non-admin user, won't this leave orphaned notes? Should this be set to always true?

Reply:
This method returns all of the recent activity that the requesting user is permitted to see. If the user is demoted, I would venture to say that they no longer have the right to view the message they created as an admin.